.travis.yml

```diff
@@ -2,7 +2,7 @@ language: c
 sudo: required
 dist: xenial
 install:
-    - sudo apt-get install -y cmake python3 python3-pip python3-setuptools pkg-config libcapstone-dev cppcheck curl wget check
+    - sudo apt-get install -y cmake python3 python3-pip python3-setuptools pkg-config libcapstone-dev cppcheck curl wget check clang-tools-6.0
```

Comment:
can you not install clang-tool

EDIT: OK, just saw it was not available on xenial... and only in the "update" repo.

You can also add PPA if you want a recent version ;-)
https://apt.llvm.org/